Index: setup/sql/mysql40/update/update09-for2.13.8-alter-c_member_pre-add-column-is_sns_entry_confirm.sql
===================================================================
--- setup/sql/mysql40/update/update09-for2.13.8-alter-c_member_pre-add-column-is_sns_entry_confirm.sql	(.../OpenPNE-2.14.0)	(リビジョン 12654)
+++ setup/sql/mysql40/update/update09-for2.13.8-alter-c_member_pre-add-column-is_sns_entry_confirm.sql	(.../OpenPNE-2.14.1)	(リビジョン 12654)
@@ -1 +1 @@
-ALTER TABLE c_member_pre ADD COLUMN is_sns_entry_confirm` tinyint(1) NOT NULL default '0';
+ALTER TABLE c_member_pre ADD COLUMN is_sns_entry_confirm tinyint(1) NOT NULL default '0';
Index: setup/sql/mysql40/upgrade/upgrade-2.12to2.14.sql
===================================================================
--- setup/sql/mysql40/upgrade/upgrade-2.12to2.14.sql	(.../OpenPNE-2.14.0)	(リビジョン 12654)
+++ setup/sql/mysql40/upgrade/upgrade-2.12to2.14.sql	(.../OpenPNE-2.14.1)	(リビジョン 12654)
@@ -29,7 +29,7 @@
 ALTER TABLE c_diary ADD COLUMN is_comment_input tinyint(1) NOT NULL default 1;
 
 -- update09
-ALTER TABLE c_member_pre ADD COLUMN is_sns_entry_confirm` tinyint(1) NOT NULL default '0';
+ALTER TABLE c_member_pre ADD COLUMN is_sns_entry_confirm tinyint(1) NOT NULL default '0';
 
 -- update10
 CREATE TABLE `c_image_size` (
Index: setup/script/update/config.inc.php
===================================================================
--- setup/script/update/config.inc.php	(.../OpenPNE-2.14.0)	(リビジョン 0)
+++ setup/script/update/config.inc.php	(.../OpenPNE-2.14.1)	(リビジョン 12654)
@@ -0,0 +1,10 @@
+<?php
+/**
+ * @copyright 2005-2008 OpenPNE Project
+ * @license   http://www.php.net/license/3_01.txt PHP License 3.01
+ */
+
+define('OPENPNE_DIR', realpath('../../../'));
+require_once OPENPNE_DIR . '/config.php';
+
+?>
Index: setup/script/update/update01-for2.13.8-insert_image_size.php
===================================================================
--- setup/script/update/update01-for2.13.8-insert_image_size.php	(.../OpenPNE-2.14.0)	(リビジョン 12654)
+++ setup/script/update/update01-for2.13.8-insert_image_size.php	(.../OpenPNE-2.14.1)	(リビジョン 12654)
@@ -1,10 +1,11 @@
 <?php
-chdir("../../../public_html");
-require_once("./config.inc.php");
-require_once(OPENPNE_WEBAPP_DIR . "/init.inc");
+require_once './config.inc.php';
+chdir(OPENPNE_PUBLIC_HTML_DIR);
+require_once OPENPNE_WEBAPP_DIR . '/init.inc';
 
-//----------
-//- 登録済み画像のファイルサイズを求める
+/**
+ * 登録済み画像のファイルサイズを求める
+ */
 function get_image_size($category, $params, $handle)
 {
     // request
@@ -13,7 +14,7 @@
     $c_member_id = $params[2];
 
     // c_image 参照先
-    $image_dsn = array('main','image');
+    $image_dsn = array('main', 'image');
     foreach ($image_dsn as $key => $dsn) {
         if (!isset($GLOBALS['_OPENPNE_DSN_LIST'][$dsn])) {
             continue;
@@ -21,27 +22,24 @@
         $imagedb =& db_get_instance($dsn);
 
         // 取り出し
-        $sql  = "SELECT ";
-        if ($table <> "biz_shisetsu") {
-            $sql .= " $c_member_id as c_member_id, ";
+        $sql  = 'SELECT ';
+        if ($table <> 'biz_shisetsu') {
+            $sql .= $c_member_id . ' as c_member_id, ';
         }
-        $sql .= "$filename as image_filename ";
-        if ($table == "c_member") {
-            $sql .= ",r_date ";
-        } elseif ($table <> "biz_group" && $table <> "biz_shisetsu") {
-            $sql .= ",r_datetime ";
+        $sql .= $filename . ' as image_filename ';
+        if ($table == 'c_member') {
+            $sql .= ',r_date ';
+        } elseif ($table <> 'biz_group' && $table <> 'biz_shisetsu') {
+            $sql .= ',r_datetime ';
         }
-        $sql .= "FROM ";
-        $sql .= " $table ";
-        $sql .= "WHERE ";
-        $sql .= " $filename <> '' ";
-        $sql .= "AND ";
-        $sql .= " $filename is not NULL";
+        $sql .= 'FROM ' . $table;
+        $sql .= ' WHERE ' . $filename . ' <> ""';
+        $sql .= ' AND ' . $filename . ' is not NULL';
         $data_list = db_get_all($sql);
 
         $lines = 0;
         if ($data_list) {
-            fwrite($handle, "INSERT INTO c_image_size VALUES");
+            fwrite($handle, 'INSERT INTO c_image_size VALUES');
         }
         foreach($data_list as $data) {
             // get image size
@@ -53,28 +51,28 @@
                 $filesize = strlen(base64_decode($c_image['bin']));
             }
 
-            $ins_data = "";
+            $ins_data = '';
             if ($lines) {
-                $ins_data .= ",";
+                $ins_data .= ',';
             }
-            $ins_data .= "(";
+            $ins_data .= '(';
             if ($GLOBALS['_OPENPNE_DSN_LIST']['main']['dsn']['phptype'] == 'pgsql') {
                 $ins_data .= " nextval('c_image_size_c_image_size_id_seq')";
             } else {
-                $ins_data .= " null";
+                $ins_data .= ' null';
             }
             $ins_data .= ",'" . $data['image_filename'] . "'";
             $ins_data .= ",'" . $data['c_member_id'] . "'";
             $ins_data .= ",'" . $filesize . "'";
             $ins_data .= ",'" . $category . "'";
-            if ($table == "biz_group" || $table == "biz_shisetsu") {
+            if ($table == 'biz_group' || $table == 'biz_shisetsu') {
                 $ins_data .= ",'" . db_now() . "'";
-            } elseif ($table == "c_member") {
+            } elseif ($table == 'c_member') {
                 $ins_data .= ",'" . $data['r_date'] . "'";
             } else {
                 $ins_data .= ",'" . $data['r_datetime'] . "'";
             }
-            $ins_data .= ")";
+            $ins_data .= ')';
             fwrite($handle, $ins_data);
             $lines ++;
         }
@@ -85,12 +83,13 @@
     
 }
 
-//----------
-//- テンポラリファイルを作成
+/**
+ * テンポラリファイルを作成
+ */
 function open_temp_file($mode, $filename = '')
 {
     if (!$filename) {
-        $w_filename = tempnam(OPENPNE_VAR_DIR . "/tmp", "image_size");
+        $w_filename = tempnam(OPENPNE_VAR_DIR . '/tmp', 'image_size');
     } else {
         $w_filename = $filename;
     }
@@ -99,11 +98,12 @@
     return array($w_filename, $handle);
 }
 
-//----------
-//- load data infile
+/**
+ * load data infile
+ */
 function load_data_infile($filename)
 {
-    list($r_filename, $r_handle) = open_temp_file("r", $filename);
+    list($r_filename, $r_handle) = open_temp_file('r', $filename);
     while (!feof($r_handle)) {
         $sql = fgets($r_handle);
         if ($sql) db_query($sql);
@@ -111,8 +111,9 @@
     close_temp_file($filename, $r_handle);
 }
 
-//----------
-//- テンポラリを閉じる
+/**
+ * テンポラリを閉じる
+ */
 function close_temp_file($filename, $handle)
 {
     fclose($handle);
@@ -120,46 +121,44 @@
 }
 
 
-//----------
-//- 処理本体
-//----------
-//  biz_shisetsuは、member_idを持っていないため、設定不可
-
-// array([category] => array([table],[filename field],[c_member_id field]))
-
+/**
+ * 処理本体
+ * biz_shisetsuは、member_idを持っていないため、設定不可
+ * array([category] => array([table],[filename field],[c_member_id field])
+ */
 $category_list = array(
-          'album' => array(
-                         array('c_album_image', 'image_filename', 'c_member_id')
-                          ),
-          'commu' => array(
-                         array('c_commu_topic_comment', 'image_filename1', 'c_member_id')
-                        ,array('c_commu_topic_comment', 'image_filename2', 'c_member_id')
-                        ,array('c_commu_topic_comment', 'image_filename3', 'c_member_id')
-                          ),
-          'diary' => array(
-                         array('c_diary', 'image_filename_1', 'c_member_id')
-                        ,array('c_diary', 'image_filename_2', 'c_member_id')
-                        ,array('c_diary', 'image_filename_3', 'c_member_id')
-                        ,array('c_diary_comment', 'image_filename_1', 'c_member_id')
-                        ,array('c_diary_comment', 'image_filename_2', 'c_member_id')
-                        ,array('c_diary_comment', 'image_filename_3', 'c_member_id')
-                        ,array('c_album', 'album_cover_image', 'c_member_id')
-                          ),
-           'other' => array(
-                         array('c_commu', 'image_filename', 'c_member_id_admin')
-                        ,array('biz_group', 'image_filename', 'admin_id')
-                        ,array('biz_shisetsu', 'image_filename', 'c_member_id')
-                        ,array('c_member', 'image_filename_1', 'c_member_id')
-                        ,array('c_member', 'image_filename_2', 'c_member_id')
-                        ,array('c_member', 'image_filename_3', 'c_member_id')
-                        ,array('c_message', 'image_filename_1', 'c_member_id_from')
-                        ,array('c_message', 'image_filename_2', 'c_member_id_from')
-                        ,array('c_message', 'image_filename_3', 'c_member_id_from')
-                          )
-                       );
+    'album' => array(
+        array('c_album_image', 'image_filename', 'c_member_id')
+    ),
+    'commu' => array(
+        array('c_commu_topic_comment', 'image_filename1', 'c_member_id'),
+        array('c_commu_topic_comment', 'image_filename2', 'c_member_id'),
+        array('c_commu_topic_comment', 'image_filename3', 'c_member_id'),
+    ),
+    'diary' => array(
+        array('c_diary', 'image_filename_1', 'c_member_id'),
+        array('c_diary', 'image_filename_2', 'c_member_id'),
+        array('c_diary', 'image_filename_3', 'c_member_id'),
+        array('c_diary_comment', 'image_filename_1', 'c_member_id'),
+        array('c_diary_comment', 'image_filename_2', 'c_member_id'),
+        array('c_diary_comment', 'image_filename_3', 'c_member_id'),
+        array('c_album', 'album_cover_image', 'c_member_id'),
+    ),
+     'other' => array(
+        array('c_commu', 'image_filename', 'c_member_id_admin'),
+        array('biz_group', 'image_filename', 'admin_id'),
+        array('biz_shisetsu', 'image_filename', 'c_member_id'),
+        array('c_member', 'image_filename_1', 'c_member_id'),
+        array('c_member', 'image_filename_2', 'c_member_id'),
+        array('c_member', 'image_filename_3', 'c_member_id'),
+        array('c_message', 'image_filename_1', 'c_member_id_from'),
+        array('c_message', 'image_filename_2', 'c_member_id_from'),
+        array('c_message', 'image_filename_3', 'c_member_id_from'),
+    ),
+);
 
 // Insert格納用テンポラリファイル作成
-list($filename, $handle) = open_temp_file("w");
+list($filename, $handle) = open_temp_file('w');
 
 //テーブル分ループ
 foreach ($category_list as $category => $table_list) {
@@ -177,8 +176,7 @@
 // Insert実行
 load_data_infile($filename);
 
-// テンポラリファイル削除
-//unlink($filename);
+echo 'Complete!';
 ?>
 
 
Index: setup/script/upgrade/config.inc.php
===================================================================
--- setup/script/upgrade/config.inc.php	(.../OpenPNE-2.14.0)	(リビジョン 0)
+++ setup/script/upgrade/config.inc.php	(.../OpenPNE-2.14.1)	(リビジョン 12654)
@@ -0,0 +1,10 @@
+<?php
+/**
+ * @copyright 2005-2008 OpenPNE Project
+ * @license   http://www.php.net/license/3_01.txt PHP License 3.01
+ */
+
+define('OPENPNE_DIR', realpath('../../../'));
+require_once OPENPNE_DIR . '/config.php';
+
+?>
Index: setup/script/upgrade/upgrade01.php
===================================================================
--- setup/script/upgrade/upgrade01.php	(.../OpenPNE-2.14.0)	(リビジョン 12654)
+++ setup/script/upgrade/upgrade01.php	(.../OpenPNE-2.14.1)	(リビジョン 12654)
@@ -1,10 +1,11 @@
 <?php
-chdir("../../../public_html");
-require_once("./config.inc.php");
-require_once(OPENPNE_WEBAPP_DIR . "/init.inc");
+require_once './config.inc.php';
+chdir(OPENPNE_PUBLIC_HTML_DIR);
+require_once OPENPNE_WEBAPP_DIR . '/init.inc';
 
-//----------
-//- 登録済み画像のファイルサイズを求める
+/**
+ * 登録済み画像のファイルサイズを求める
+ */
 function get_image_size($category, $params, $handle)
 {
     // request
@@ -13,7 +14,7 @@
     $c_member_id = $params[2];
 
     // c_image 参照先
-    $image_dsn = array('main','image');
+    $image_dsn = array('main', 'image');
     foreach ($image_dsn as $key => $dsn) {
         if (!isset($GLOBALS['_OPENPNE_DSN_LIST'][$dsn])) {
             continue;
@@ -21,27 +22,24 @@
         $imagedb =& db_get_instance($dsn);
 
         // 取り出し
-        $sql  = "SELECT ";
-        if ($table <> "biz_shisetsu") {
-            $sql .= " $c_member_id as c_member_id, ";
+        $sql  = 'SELECT ';
+        if ($table <> 'biz_shisetsu') {
+            $sql .= $c_member_id . ' as c_member_id, ';
         }
-        $sql .= "$filename as image_filename ";
-        if ($table == "c_member") {
-            $sql .= ",r_date ";
-        } elseif ($table <> "biz_group" && $table <> "biz_shisetsu") {
-            $sql .= ",r_datetime ";
+        $sql .= $filename . ' as image_filename ';
+        if ($table == 'c_member') {
+            $sql .= ',r_date ';
+        } elseif ($table <> 'biz_group' && $table <> 'biz_shisetsu') {
+            $sql .= ',r_datetime ';
         }
-        $sql .= "FROM ";
-        $sql .= " $table ";
-        $sql .= "WHERE ";
-        $sql .= " $filename <> '' ";
-        $sql .= "AND ";
-        $sql .= " $filename is not NULL";
+        $sql .= 'FROM ' . $table;
+        $sql .= ' WHERE ' . $filename . ' <> ""';
+        $sql .= ' AND ' . $filename . ' is not NULL';
         $data_list = db_get_all($sql);
 
         $lines = 0;
         if ($data_list) {
-            fwrite($handle, "INSERT INTO c_image_size VALUES");
+            fwrite($handle, 'INSERT INTO c_image_size VALUES');
         }
         foreach($data_list as $data) {
             // get image size
@@ -53,28 +51,28 @@
                 $filesize = strlen(base64_decode($c_image['bin']));
             }
 
-            $ins_data = "";
+            $ins_data = '';
             if ($lines) {
-                $ins_data .= ",";
+                $ins_data .= ',';
             }
-            $ins_data .= "(";
+            $ins_data .= '(';
             if ($GLOBALS['_OPENPNE_DSN_LIST']['main']['dsn']['phptype'] == 'pgsql') {
                 $ins_data .= " nextval('c_image_size_c_image_size_id_seq')";
             } else {
-                $ins_data .= " null";
+                $ins_data .= ' null';
             }
             $ins_data .= ",'" . $data['image_filename'] . "'";
             $ins_data .= ",'" . $data['c_member_id'] . "'";
             $ins_data .= ",'" . $filesize . "'";
             $ins_data .= ",'" . $category . "'";
-            if ($table == "biz_group" || $table == "biz_shisetsu") {
+            if ($table == 'biz_group' || $table == 'biz_shisetsu') {
                 $ins_data .= ",'" . db_now() . "'";
-            } elseif ($table == "c_member") {
+            } elseif ($table == 'c_member') {
                 $ins_data .= ",'" . $data['r_date'] . "'";
             } else {
                 $ins_data .= ",'" . $data['r_datetime'] . "'";
             }
-            $ins_data .= ")";
+            $ins_data .= ')';
             fwrite($handle, $ins_data);
             $lines ++;
         }
@@ -85,12 +83,13 @@
     
 }
 
-//----------
-//- テンポラリファイルを作成
+/**
+ * テンポラリファイルを作成
+ */
 function open_temp_file($mode, $filename = '')
 {
     if (!$filename) {
-        $w_filename = tempnam(OPENPNE_VAR_DIR . "/tmp", "image_size");
+        $w_filename = tempnam(OPENPNE_VAR_DIR . '/tmp', 'image_size');
     } else {
         $w_filename = $filename;
     }
@@ -99,11 +98,12 @@
     return array($w_filename, $handle);
 }
 
-//----------
-//- load data infile
+/**
+ * load data infile
+ */
 function load_data_infile($filename)
 {
-    list($r_filename, $r_handle) = open_temp_file("r", $filename);
+    list($r_filename, $r_handle) = open_temp_file('r', $filename);
     while (!feof($r_handle)) {
         $sql = fgets($r_handle);
         if ($sql) db_query($sql);
@@ -111,8 +111,9 @@
     close_temp_file($filename, $r_handle);
 }
 
-//----------
-//- テンポラリを閉じる
+/**
+ * テンポラリを閉じる
+ */
 function close_temp_file($filename, $handle)
 {
     fclose($handle);
@@ -120,46 +121,44 @@
 }
 
 
-//----------
-//- 処理本体
-//----------
-//  biz_shisetsuは、member_idを持っていないため、設定不可
-
-// array([category] => array([table],[filename field],[c_member_id field]))
-
+/**
+ * 処理本体
+ * biz_shisetsuは、member_idを持っていないため、設定不可
+ * array([category] => array([table],[filename field],[c_member_id field])
+ */
 $category_list = array(
-          'album' => array(
-                         array('c_album_image', 'image_filename', 'c_member_id')
-                          ),
-          'commu' => array(
-                         array('c_commu_topic_comment', 'image_filename1', 'c_member_id')
-                        ,array('c_commu_topic_comment', 'image_filename2', 'c_member_id')
-                        ,array('c_commu_topic_comment', 'image_filename3', 'c_member_id')
-                          ),
-          'diary' => array(
-                         array('c_diary', 'image_filename_1', 'c_member_id')
-                        ,array('c_diary', 'image_filename_2', 'c_member_id')
-                        ,array('c_diary', 'image_filename_3', 'c_member_id')
-                        ,array('c_diary_comment', 'image_filename_1', 'c_member_id')
-                        ,array('c_diary_comment', 'image_filename_2', 'c_member_id')
-                        ,array('c_diary_comment', 'image_filename_3', 'c_member_id')
-                        ,array('c_album', 'album_cover_image', 'c_member_id')
-                          ),
-           'other' => array(
-                         array('c_commu', 'image_filename', 'c_member_id_admin')
-                        ,array('biz_group', 'image_filename', 'admin_id')
-                        ,array('biz_shisetsu', 'image_filename', 'c_member_id')
-                        ,array('c_member', 'image_filename_1', 'c_member_id')
-                        ,array('c_member', 'image_filename_2', 'c_member_id')
-                        ,array('c_member', 'image_filename_3', 'c_member_id')
-                        ,array('c_message', 'image_filename_1', 'c_member_id_from')
-                        ,array('c_message', 'image_filename_2', 'c_member_id_from')
-                        ,array('c_message', 'image_filename_3', 'c_member_id_from')
-                          )
-                       );
+    'album' => array(
+        array('c_album_image', 'image_filename', 'c_member_id')
+    ),
+    'commu' => array(
+        array('c_commu_topic_comment', 'image_filename1', 'c_member_id'),
+        array('c_commu_topic_comment', 'image_filename2', 'c_member_id'),
+        array('c_commu_topic_comment', 'image_filename3', 'c_member_id'),
+    ),
+    'diary' => array(
+        array('c_diary', 'image_filename_1', 'c_member_id'),
+        array('c_diary', 'image_filename_2', 'c_member_id'),
+        array('c_diary', 'image_filename_3', 'c_member_id'),
+        array('c_diary_comment', 'image_filename_1', 'c_member_id'),
+        array('c_diary_comment', 'image_filename_2', 'c_member_id'),
+        array('c_diary_comment', 'image_filename_3', 'c_member_id'),
+        array('c_album', 'album_cover_image', 'c_member_id'),
+    ),
+     'other' => array(
+        array('c_commu', 'image_filename', 'c_member_id_admin'),
+        array('biz_group', 'image_filename', 'admin_id'),
+        array('biz_shisetsu', 'image_filename', 'c_member_id'),
+        array('c_member', 'image_filename_1', 'c_member_id'),
+        array('c_member', 'image_filename_2', 'c_member_id'),
+        array('c_member', 'image_filename_3', 'c_member_id'),
+        array('c_message', 'image_filename_1', 'c_member_id_from'),
+        array('c_message', 'image_filename_2', 'c_member_id_from'),
+        array('c_message', 'image_filename_3', 'c_member_id_from'),
+    ),
+);
 
 // Insert格納用テンポラリファイル作成
-list($filename, $handle) = open_temp_file("w");
+list($filename, $handle) = open_temp_file('w');
 
 //テーブル分ループ
 foreach ($category_list as $category => $table_list) {
@@ -177,8 +176,7 @@
 // Insert実行
 load_data_infile($filename);
 
-// テンポラリファイル削除
-//unlink($filename);
+echo 'Complete!';
 ?>
 
 
Index: setup/OpenPNE_Setup.html
===================================================================
--- setup/OpenPNE_Setup.html	(.../OpenPNE-2.14.0)	(リビジョン 12654)
+++ setup/OpenPNE_Setup.html	(.../OpenPNE-2.14.1)	(リビジョン 12654)
@@ -61,7 +61,7 @@
 
 <h1>OpenPNE セットアップガイド</h1>
 
-<p>最終更新日: 2009/07/06</p>
+<p>最終更新日: 2009/08/10</p>
 
 <h2>目次</h2>
 <ul>
@@ -370,7 +370,9 @@
 <tr><td>コミュニティ写真変更</td><td>ci<var>XXX</var>-<var>YYY</var>@MAIL_SERVER_DOMAIN</td><td>ci<var>XXX</var>@MAIL_SERVER_DOMAIN</td></tr>
 <tr><td>トピック・イベント写真変更</td><td>ti<var>XXX</var>-<var>YYY</var>@MAIL_SERVER_DOMAIN</td><td>ti<var>XXX</var>@MAIL_SERVER_DOMAIN</td></tr>
 <tr><td>日記コメントメール投稿</td><td>bc<var>XXX</var>-<var>YYY</var>@MAIL_SERVER_DOMAIN</td><td>bc<var>XXX</var>@MAIL_SERVER_DOMAIN</td></tr>
+<tr><td>アルバム投稿</td><td>a<var>XXX</var>-<var>YYY</var>@MAIL_SERVER_DOMAIN</td><td>album@MAIL_SERVER_DOMAIN</td></tr>
 <tr><td>アルバム画像投稿</td><td>ai<var>XXX</var>-<var>YYY</var>@MAIL_SERVER_DOMAIN</td><td>ai<var>XXX</var>@MAIL_SERVER_DOMAIN</td></tr>
+<tr><td>アルバム表紙投稿</td><td>ac<var>XXX</var>-<var>YYY</var>@MAIL_SERVER_DOMAIN</td><td>ac<var>XXX</var>@MAIL_SERVER_DOMAIN</td></tr>
 </table>
 <ul>
 <li><var>XXX</var> には数字。</li>
Index: setup/OpenPNE_Setup_pgsql.html
===================================================================
--- setup/OpenPNE_Setup_pgsql.html	(.../OpenPNE-2.14.0)	(リビジョン 12654)
+++ setup/OpenPNE_Setup_pgsql.html	(.../OpenPNE-2.14.1)	(リビジョン 12654)
@@ -61,7 +61,7 @@
 
 <h1>OpenPNE セットアップガイド</h1>
 
-<p>最終更新日: 2009/07/06</p>
+<p>最終更新日: 2009/08/10</p>
 
 <h2>目次</h2>
 <ul>
@@ -333,7 +333,9 @@
 <tr><td>コミュニティ写真変更</td><td>ci<var>XXX</var>-<var>YYY</var>@MAIL_SERVER_DOMAIN</td><td>ci<var>XXX</var>@MAIL_SERVER_DOMAIN</td></tr>
 <tr><td>トピック・イベント写真変更</td><td>ti<var>XXX</var>-<var>YYY</var>@MAIL_SERVER_DOMAIN</td><td>ti<var>XXX</var>@MAIL_SERVER_DOMAIN</td></tr>
 <tr><td>日記コメントメール投稿</td><td>bc<var>XXX</var>-<var>YYY</var>@MAIL_SERVER_DOMAIN</td><td>bc<var>XXX</var>@MAIL_SERVER_DOMAIN</td></tr>
+<tr><td>アルバム投稿</td><td>a<var>XXX</var>-<var>YYY</var>@MAIL_SERVER_DOMAIN</td><td>album@MAIL_SERVER_DOMAIN</td></tr>
 <tr><td>アルバム画像投稿</td><td>ai<var>XXX</var>-<var>YYY</var>@MAIL_SERVER_DOMAIN</td><td>ai<var>XXX</var>@MAIL_SERVER_DOMAIN</td></tr>
+<tr><td>アルバム表紙投稿</td><td>ac<var>XXX</var>-<var>YYY</var>@MAIL_SERVER_DOMAIN</td><td>ac<var>XXX</var>@MAIL_SERVER_DOMAIN</td></tr>
 </table>
 <ul>
 <li><var>XXX</var> には数字。</li>
Index: setup/OpenPNE_Upgrade.html
===================================================================
--- setup/OpenPNE_Upgrade.html	(.../OpenPNE-2.14.0)	(リビジョン 12654)
+++ setup/OpenPNE_Upgrade.html	(.../OpenPNE-2.14.1)	(リビジョン 12654)
@@ -64,7 +64,7 @@
 
 <h1>OpenPNE アップグレードガイド</h1>
 
-<p>最終更新日: 2009/07/06</p>
+<p>最終更新日: 2009/08/10</p>
 
 <h2>目次</h2>
 <ul>
@@ -85,7 +85,7 @@
     <li><a href="#section2-3-2">2-3-2. OpenPNE2.14で変更された項目</a></li>
     </ul></li>
   <li><a href="#section2-4">2-4. メールサーバの設定</a></li>
-  <li><a href="section2-5">2-5. アップグレードスクリプトの実行</a></li>
+  <li><a href="#section2-5">2-5. アップグレードスクリプトの実行</a></li>
   </ul></li>
 </ul>
 
@@ -211,7 +211,9 @@
 
 <table>
 <tr><th>項目</th><th>新形式</th><th>旧形式</th></tr>
+<tr><td>アルバム投稿</td><td>a<var>XXX</var>-<var>YYY</var>@MAIL_SERVER_DOMAIN</td><td>album@MAIL_SERVER_DOMAIN</td></tr>
 <tr><td>アルバム画像投稿</td><td>ai<var>XXX</var>-<var>YYY</var>@MAIL_SERVER_DOMAIN</td><td>ai<var>XXX</var>@MAIL_SERVER_DOMAIN</td></tr>
+<tr><td>アルバム表紙投稿</td><td>ac<var>XXX</var>-<var>YYY</var>@MAIL_SERVER_DOMAIN</td><td>ac<var>XXX</var>@MAIL_SERVER_DOMAIN</td></tr>
 </table>
 <ul>
 <li><var>XXX</var> には数字。</li>
@@ -237,6 +239,8 @@
 $ cd setup/script/upgrade/
 $ php upgrade01.php
 </pre>
+<p>※スクリプトの実行は「setup/script/upgrade/」で行ってください。<br />
+「Complete!」と表示されたらスクリプト完了です。</p>
 </div>
 
 <hr>
Index: webapp/lib/mail/sns.php
===================================================================
--- webapp/lib/mail/sns.php	(.../OpenPNE-2.14.0)	(リビジョン 12654)
+++ webapp/lib/mail/sns.php	(.../OpenPNE-2.14.1)	(リビジョン 12654)
@@ -690,13 +690,13 @@
                 return false;
             }
 
-            db_image_insert_c_image($filename, $image_data);
+            db_image_insert_c_image($filename, $image_data, $filesize, $this->c_member_id);
             //アルバムの表紙に写真ファイル名を登録
             db_album_update_c_album_album_cover_image($ins_id,$filename);
         } else {
-    		$this->error_mail('写真が添付されていないか、ファイルサイズが大きすぎるため、ファイル表紙を登録できませんでした。');
-    		m_debug_log('mail_sns::add_album() no images');
-    		return false;
+            $this->error_mail('写真が添付されていないか、ファイルサイズが大きすぎるため、ファイル表紙を登録できませんでした。');
+            m_debug_log('mail_sns::add_album() no images');
+            return false;
         }
 
         return true;
@@ -739,7 +739,7 @@
         }
 
         $filename = 'a_' . $c_album_id . '_1_' . time() . '.' . $image_ext;
-        db_image_insert_c_image($filename, $image_data);
+        db_image_insert_c_image($filename, $image_data, $image_size, $this->c_member_id);
         if (!$subject) {
             // 説明文が空の場合はファイル名を挿入する
             $subject = $filename;
@@ -999,10 +999,10 @@
                 return false;
             }
 
-            db_image_insert_c_image($filename, $image_data);
+            db_image_insert_c_image($filename, $image_data, $filesize, $this->c_member_id);
             //アルバムデータの変更
             $c_album_cover = $c_album['album_cover_image'];
-            db_album_image_data_delete($c_album_cover);
+            db_album_image_data_delete($c_album_cover, $c_album['c_member_id']);
             db_album_update_c_album_album_cover_image($c_album_id,$filename);
         } else {
             $this->error_mail('写真が添付されていないか、ファイルサイズが大きすぎるため、アルバム表紙を変更できませんでした。');
Index: webapp/lib/OpenPNE/Amazon.php
===================================================================
--- webapp/lib/OpenPNE/Amazon.php	(.../OpenPNE-2.14.0)	(リビジョン 12654)
+++ webapp/lib/OpenPNE/Amazon.php	(.../OpenPNE-2.14.1)	(リビジョン 12654)
@@ -5,6 +5,7 @@
  */
 
 require_once 'Services/Amazon.php';
+require_once 'PHP/Compat/Function/mhash.php';
 
 /**
  * OpenPNEでAmazonECSを利用するためのクラス
Index: webapp/lib/db/album.php
===================================================================
--- webapp/lib/db/album.php	(.../OpenPNE-2.14.0)	(リビジョン 12654)
+++ webapp/lib/db/album.php	(.../OpenPNE-2.14.1)	(リビジョン 12654)
@@ -389,7 +389,7 @@
 }
 
 
-function db_album_update_c_album($c_album_id,$subject,$description,$public_flag,$image_filename)
+function db_album_update_c_album($c_album_id, $subject, $description, $public_flag, $image_filename = null)
 {
     $data = array(
         'subject' => $subject,
Index: webapp/lib/db/etc.php
===================================================================
--- webapp/lib/db/etc.php	(.../OpenPNE-2.14.0)	(リビジョン 12654)
+++ webapp/lib/db/etc.php	(.../OpenPNE-2.14.1)	(リビジョン 12654)
@@ -197,7 +197,10 @@
     }
 
     $storage = Auth::_factory($auth_config['storage'],$auth_config['options']);
-    return $storage->fetchData($username, $password, false);
+    if ($storage->fetchData($username, $password, false) === true) {
+        return true;
+    }
+    return false;
 }
 
 /**
Index: webapp/lib/db/image.php
===================================================================
--- webapp/lib/db/image.php	(.../OpenPNE-2.14.0)	(リビジョン 12654)
+++ webapp/lib/db/image.php	(.../OpenPNE-2.14.1)	(リビジョン 12654)
@@ -159,7 +159,7 @@
     db_query($sql, $params);
 }
 
-function db_image_data_copy($filename, $new_filename)
+function db_image_data_copy($filename, $new_filename, $c_member_id, $filesize)
 {
     if (!$filename) return false;
 
@@ -173,7 +173,7 @@
     $bin = base64_decode($c_image[0]['bin']);
     $type = $c_image[0]['type'];
 
-    return db_image_insert_c_image($new_filename, $bin, $type);
+    return db_image_insert_c_image($new_filename, $bin, $filesize, $c_member_id, null, $type);
 }
 
 /**
Index: webapp/lib/ktaiIP.php
===================================================================
--- webapp/lib/ktaiIP.php	(.../OpenPNE-2.14.0)	(リビジョン 12654)
+++ webapp/lib/ktaiIP.php	(.../OpenPNE-2.14.1)	(リビジョン 12654)
@@ -36,7 +36,7 @@
 '118.159.133.0/25',
 '118.159.132.160/27',
 
-// SoftBank (2008/2/29更新)
+// SoftBank (2009/8/10更新)
 // http://creation.mb.softbank.jp/web/web_ip.html
 '123.108.236.0/24',
 '123.108.237.0/27',
@@ -45,9 +45,7 @@
 '210.146.7.192/26',
 '210.146.60.192/26',
 '210.151.9.128/26',
-'210.169.130.112/28',
 '210.175.1.128/25',
-'210.228.189.0/24',
 '211.8.159.128/25',
 
 // WILLCOM (2009/4/2更新)
Index: webapp/modules/setup/do/setup.php
===================================================================
--- webapp/modules/setup/do/setup.php	(.../OpenPNE-2.14.0)	(リビジョン 12654)
+++ webapp/modules/setup/do/setup.php	(.../OpenPNE-2.14.1)	(リビジョン 12654)
@@ -30,7 +30,8 @@
         if (OPENPNE_AUTH_MODE == 'slavepne') {
             $auth_config = get_auth_config(false);
             $storage = Auth::_factory($auth_config['storage'],$auth_config['options']);
-            if (!$storage->fetchData($requests['username'], $requests['password'], false)){
+            $result = $storage->fetchData($requests['username'], $requests['password'], false);
+            if ($result !== true) {
                 $errors[] = 'ログインIDまたはパスワードが一致しません';
             }
         }
Index: webapp/modules/pc/do/h_album_cover_edit_insert_c_album.php
===================================================================
--- webapp/modules/pc/do/h_album_cover_edit_insert_c_album.php	(.../OpenPNE-2.14.0)	(リビジョン 12654)
+++ webapp/modules/pc/do/h_album_cover_edit_insert_c_album.php	(.../OpenPNE-2.14.1)	(リビジョン 12654)
@@ -61,7 +61,7 @@
         $c_album_cover = $c_album['album_cover_image'];
 
         if ($tmpfile_1) {
-            db_album_image_data_delete($c_album_cover);
+            db_album_image_data_delete($c_album_cover, $u);
             $filename_1 = image_insert_c_image4tmp("a_{$target_c_album_id}_1", $tmpfile_1, $u, 'other');
         }
 
Index: webapp/modules/pc/do/h_album_image_edit_insert_c_album_image.php
===================================================================
--- webapp/modules/pc/do/h_album_image_edit_insert_c_album_image.php	(.../OpenPNE-2.14.0)	(リビジョン 12654)
+++ webapp/modules/pc/do/h_album_image_edit_insert_c_album_image.php	(.../OpenPNE-2.14.1)	(リビジョン 12654)
@@ -74,7 +74,7 @@
             t_image_clear_tmp($sessid);
 
             if (!db_album_is_insertable4c_member_id($u, $filesize - $c_album_image['filesize'])) {
-                db_album_image_data_delete($filename);
+                db_album_image_data_delete($filename, $u);
 
                 $msg = 'これ以上写真を投稿することができません。';
                 if (!db_album_is_insertable4c_member_id($u)) {
@@ -90,7 +90,7 @@
                 openpne_redirect('pc', 'page_h_album_image_edit', $p);
             }
 
-            db_album_image_data_delete($c_album_image['image_filename']);
+            db_album_image_data_delete($c_album_image['image_filename'], $u);
         }
 
         db_album_update_c_album_image($target_c_album_image_id, $filename, $image_description, $filesize);
Index: webapp/modules/pc/templates/h_message.tpl
===================================================================
--- webapp/modules/pc/templates/h_message.tpl	(.../OpenPNE-2.14.0)	(リビジョン 12654)
+++ webapp/modules/pc/templates/h_message.tpl	(.../OpenPNE-2.14.1)	(リビジョン 12654)
@@ -65,7 +65,7 @@
 
 ({if $c_message.filename && $smarty.const.OPENPNE_USE_FILEUPLOAD})
 <div class="block attachFile"><ul>
-<li><a href="({t_url m=pc a=do_h_message_file_download})&amp;target_c_message_id=({$c_message.c_message_id})&amp;sessid=({$PHPSESSID})">({$c_message.original_filename})</a></li>
+<li><a href="({t_url m=pc a=do_h_message_file_download})&amp;target_c_message_id=({$c_message.c_message_id})&amp;sessid=({$PHPSESSID})&amp;({$smarty.now})">({$c_message.original_filename})</a></li>
 </ul></div>
 ({/if})
 
Index: webapp/modules/pc/templates/c_topic_detail.tpl
===================================================================
--- webapp/modules/pc/templates/c_topic_detail.tpl	(.../OpenPNE-2.14.0)	(リビジョン 12654)
+++ webapp/modules/pc/templates/c_topic_detail.tpl	(.../OpenPNE-2.14.1)	(リビジョン 12654)
@@ -31,7 +31,7 @@
 </div>
 ({if $c_topic.filename && $smarty.const.OPENPNE_USE_FILEUPLOAD})
 <div class="block attachFile"><ul>
-<li><a href="({t_url m=pc a=do_c_file_download})&amp;target_c_commu_topic_id=({$c_topic.c_commu_topic_id})&amp;sessid=({$PHPSESSID})">({$c_topic.original_filename})</a></li>
+<li><a href="({t_url m=pc a=do_c_file_download})&amp;target_c_commu_topic_id=({$c_topic.c_commu_topic_id})&amp;sessid=({$PHPSESSID})&amp;({$smarty.now})">({$c_topic.original_filename})</a></li>
 </ul></div>
 ({/if})
 </dd>
@@ -113,7 +113,7 @@
 </div>
 ({if $item.filename && $smarty.const.OPENPNE_USE_FILEUPLOAD})
 <div class="block attachFile"><ul>
-<li><a href="({t_url m=pc a=do_c_file_download})&amp;target_c_commu_topic_comment_id=({$item.c_commu_topic_comment_id})&amp;sessid=({$PHPSESSID})">({$item.original_filename})</a></li>
+<li><a href="({t_url m=pc a=do_c_file_download})&amp;target_c_commu_topic_comment_id=({$item.c_commu_topic_comment_id})&amp;sessid=({$PHPSESSID})&amp;({$smarty.now})">({$item.original_filename})</a></li>
 </ul></div>
 ({/if})
 </dd>
Index: webapp/modules/pc/templates/c_topic_write_delete_confirm.tpl
===================================================================
--- webapp/modules/pc/templates/c_topic_write_delete_confirm.tpl	(.../OpenPNE-2.14.0)	(リビジョン 12654)
+++ webapp/modules/pc/templates/c_topic_write_delete_confirm.tpl	(.../OpenPNE-2.14.1)	(リビジョン 12654)
@@ -16,7 +16,7 @@
 </td></tr>
 ({if $smarty.const.OPENPNE_USE_FILEUPLOAD})
 ({if $c_commu_topic_comment.filename})
-<tr><th>ファイル</th><td><a href="({t_url m=pc a=do_c_file_download})&amp;target_c_commu_topic_comment_id=({$c_commu_topic_comment.c_commu_topic_comment_id})&amp;sessid=({$PHPSESSID})">({$c_commu_topic_comment.original_filename})</a></td></tr>
+<tr><th>ファイル</th><td><a href="({t_url m=pc a=do_c_file_download})&amp;target_c_commu_topic_comment_id=({$c_commu_topic_comment.c_commu_topic_comment_id})&amp;sessid=({$PHPSESSID})&amp;({$smarty.now})">({$c_commu_topic_comment.original_filename})</a></td></tr>
 ({/if})
 ({/if})
 </table>
Index: webapp/modules/pc/templates/c_event_detail.tpl
===================================================================
--- webapp/modules/pc/templates/c_event_detail.tpl	(.../OpenPNE-2.14.0)	(リビジョン 12654)
+++ webapp/modules/pc/templates/c_event_detail.tpl	(.../OpenPNE-2.14.1)	(リビジョン 12654)
@@ -50,7 +50,7 @@
 ({if $c_topic.filename && $smarty.const.OPENPNE_USE_FILEUPLOAD})
 <th>ファイル</th>
 <td>
-<a href="({t_url m=pc a=do_c_file_download})&amp;target_c_commu_topic_id=({$c_topic.c_commu_topic_id})&amp;sessid=({$PHPSESSID})">({$c_topic.original_filename})</a></td>
+<a href="({t_url m=pc a=do_c_file_download})&amp;target_c_commu_topic_id=({$c_topic.c_commu_topic_id})&amp;sessid=({$PHPSESSID})&amp;({$smarty.now})">({$c_topic.original_filename})</a></td>
 </tr><tr>
 ({/if})
 <th>募集期限</th>
@@ -165,7 +165,7 @@
 </div>
 ({if $item.filename && $smarty.const.OPENPNE_USE_FILEUPLOAD})
 <div class="block attachFile"><ul>
-<li><a href="({t_url m=pc a=do_c_file_download})&amp;target_c_commu_topic_comment_id=({$item.c_commu_topic_comment_id})&amp;sessid=({$PHPSESSID})">({$item.original_filename})</a></li>
+<li><a href="({t_url m=pc a=do_c_file_download})&amp;target_c_commu_topic_comment_id=({$item.c_commu_topic_comment_id})&amp;sessid=({$PHPSESSID})&amp;({$smarty.now})">({$item.original_filename})</a></li>
 </ul></div>
 ({/if})
 </dd>
Index: webapp/modules/admin/templates/download_xml.tpl
===================================================================
--- webapp/modules/admin/templates/download_xml.tpl	(.../OpenPNE-2.14.0)	(リビジョン 12654)
+++ webapp/modules/admin/templates/download_xml.tpl	(.../OpenPNE-2.14.1)	(リビジョン 12654)
@@ -25,6 +25,7 @@
 <input type="hidden" name="m" value="({$module_name})" />
 <input type="hidden" name="a" value="do_({$hash_tbl->hash('download_xml','do')})" />
 <input type="hidden" name="sessid" value="({$PHPSESSID})" />
+<input type="hidden" name="timestamp" value="({$smarty.now})" />
 
 <table class="basicType2">
 <tr class="cell01">
Index: webapp/modules/admin/templates/access_analysis_page.tpl
===================================================================
--- webapp/modules/admin/templates/access_analysis_page.tpl	(.../OpenPNE-2.14.0)	(リビジョン 12654)
+++ webapp/modules/admin/templates/access_analysis_page.tpl	(.../OpenPNE-2.14.1)	(リビジョン 12654)
@@ -91,6 +91,7 @@
 <input type="hidden" name="ymd" value="({$ymd})" />
 <input type="hidden" name="month_flag" value="({$month_flag})" />
 <input type="hidden" name="orderby" value="({$orderby})" />
+<input type="hidden" name="timestamp" value="({$smarty.now})" />
 <p class="textBtn"><input type="submit" value="ダウンロード" /></p>
 </form>
 
Index: webapp/modules/admin/templates/access_analysis_month.tpl
===================================================================
--- webapp/modules/admin/templates/access_analysis_month.tpl	(.../OpenPNE-2.14.0)	(リビジョン 12654)
+++ webapp/modules/admin/templates/access_analysis_month.tpl	(.../OpenPNE-2.14.1)	(リビジョン 12654)
@@ -57,7 +57,8 @@
 <input type="hidden" name="a" value="do_({$hash_tbl->hash('csv_access_analysis_month','do')})" />
 <input type="hidden" name="sessid" value="({$PHPSESSID})" />
 <input type="hidden" name="ktai_flag" value="({$ktai_flag})" />
+<input type="hidden" name="timestamp" value="({$smarty.now})" />
 <p class="textBtn"><input type="submit" value="ダウンロード" /></p>
 </form>
 
-({$inc_footer|smarty:nodefaults})
\ ファイルの末尾に改行がありません
+({$inc_footer|smarty:nodefaults})
Index: webapp/modules/admin/templates/access_analysis_target_commu.tpl
===================================================================
--- webapp/modules/admin/templates/access_analysis_target_commu.tpl	(.../OpenPNE-2.14.0)	(リビジョン 12654)
+++ webapp/modules/admin/templates/access_analysis_target_commu.tpl	(.../OpenPNE-2.14.1)	(リビジョン 12654)
@@ -67,6 +67,7 @@
 <input type="hidden" name="month_flag" value="({$month_flag})" />
 <input type="hidden" name="page_name" value="({$requests.page_name})" />
 <input type="hidden" name="orderby" value="({$orderby})" />
+<input type="hidden" name="timestamp" value="({$smarty.now})" />
 <p class="textBtn"><input type="submit" value="ダウンロード" /></p>
 </form>
 ({else})
Index: webapp/modules/admin/templates/image_analysis_category.tpl
===================================================================
--- webapp/modules/admin/templates/image_analysis_category.tpl	(.../OpenPNE-2.14.0)	(リビジョン 12654)
+++ webapp/modules/admin/templates/image_analysis_category.tpl	(.../OpenPNE-2.14.1)	(リビジョン 12654)
@@ -57,6 +57,7 @@
 <input type="hidden" name="m" value="({$module_name})" />
 <input type="hidden" name="a" value="do_({$hash_tbl->hash('csv_analysis_image_category','do')})" />
 <input type="hidden" name="sessid" value="({$PHPSESSID})" />
+<input type="hidden" name="timestamp" value="({$smarty.now})" />
 <p class="textBtn"><input type="submit" value="ダウンロード" /></p>
 </form>
 
Index: webapp/modules/admin/templates/access_analysis_day.tpl
===================================================================
--- webapp/modules/admin/templates/access_analysis_day.tpl	(.../OpenPNE-2.14.0)	(リビジョン 12654)
+++ webapp/modules/admin/templates/access_analysis_day.tpl	(.../OpenPNE-2.14.1)	(リビジョン 12654)
@@ -47,6 +47,7 @@
 <input type="hidden" name="sessid" value="({$PHPSESSID})" />
 <input type="hidden" name="ktai_flag" value="({$ktai_flag})" />
 <input type="hidden" name="ymd" value="({$access_analysis_day[0].ymd})" />
+<input type="hidden" name="timestamp" value="({$smarty.now})" />
 <p class="textBtn"><input type="submit" value="ダウンロード" /></p>
 </form>
 ({$inc_footer|smarty:nodefaults})
Index: webapp/modules/admin/templates/delete_topic_selected.tpl
===================================================================
--- webapp/modules/admin/templates/delete_topic_selected.tpl	(.../OpenPNE-2.14.0)	(リビジョン 12654)
+++ webapp/modules/admin/templates/delete_topic_selected.tpl	(.../OpenPNE-2.14.1)	(リビジョン 12654)
@@ -92,7 +92,7 @@
 <tr>
 <th>ファイル</th>
 <td class="textbody">
-<a href="./?m=({$module_name})&amp;a=do_({$hash_tbl->hash('file_download','do')})&amp;filename=({$topic.filename})&amp;sessid=({$PHPSESSID})">
+<a href="./?m=({$module_name})&amp;a=do_({$hash_tbl->hash('file_download','do')})&amp;filename=({$topic.filename})&amp;sessid=({$PHPSESSID})&amp;({$smarty.now})">
 ({$topic.original_filename})
 </a>
 </td>
Index: webapp/modules/admin/templates/topic_list.tpl
===================================================================
--- webapp/modules/admin/templates/topic_list.tpl	(.../OpenPNE-2.14.0)	(リビジョン 12654)
+++ webapp/modules/admin/templates/topic_list.tpl	(.../OpenPNE-2.14.1)	(リビジョン 12654)
@@ -135,7 +135,7 @@
 <tr>
 <th>ファイル</th>
 <td class="textbody">
-<a href="./?m=({$module_name})&amp;a=do_({$hash_tbl->hash('file_download','do')})&amp;filename=({$item.filename})&amp;sessid=({$PHPSESSID})">
+<a href="./?m=({$module_name})&amp;a=do_({$hash_tbl->hash('file_download','do')})&amp;filename=({$item.filename})&amp;sessid=({$PHPSESSID})&amp;({$smarty.now})">
 ({$item.original_filename})
 </a>
 </td>
Index: webapp/modules/admin/templates/delete_topic_comment.tpl
===================================================================
--- webapp/modules/admin/templates/delete_topic_comment.tpl	(.../OpenPNE-2.14.0)	(リビジョン 12654)
+++ webapp/modules/admin/templates/delete_topic_comment.tpl	(.../OpenPNE-2.14.1)	(リビジョン 12654)
@@ -75,7 +75,7 @@
 <tr>
 <th>ファイル</th>
 <td class="textbody">
-<a href="?m=({$module_name})&amp;a=do_({$hash_tbl->hash('file_download','do')})&amp;filename=({$topic_comment.filename})&amp;sessid=({$PHPSESSID})">
+<a href="?m=({$module_name})&amp;a=do_({$hash_tbl->hash('file_download','do')})&amp;filename=({$topic_comment.filename})&amp;sessid=({$PHPSESSID})&amp;({$smarty.now})">
 ({$topic_comment.original_filename})
 </a>
 </td>
Index: webapp/modules/admin/templates/delete_topic.tpl
===================================================================
--- webapp/modules/admin/templates/delete_topic.tpl	(.../OpenPNE-2.14.0)	(リビジョン 12654)
+++ webapp/modules/admin/templates/delete_topic.tpl	(.../OpenPNE-2.14.1)	(リビジョン 12654)
@@ -91,7 +91,7 @@
 <tr>
 <th>ファイル</th>
 <td class="textbody">
-<a href="./?m=({$module_name})&amp;a=do_({$hash_tbl->hash('file_download','do')})&amp;filename=({$topic.filename})&amp;sessid=({$PHPSESSID})">
+<a href="./?m=({$module_name})&amp;a=do_({$hash_tbl->hash('file_download','do')})&amp;filename=({$topic.filename})&amp;sessid=({$PHPSESSID})&amp;({$smarty.now})">
 ({$topic.original_filename})
 </a>
 </td>
Index: webapp/modules/admin/templates/access_analysis_target_member.tpl
===================================================================
--- webapp/modules/admin/templates/access_analysis_target_member.tpl	(.../OpenPNE-2.14.0)	(リビジョン 12654)
+++ webapp/modules/admin/templates/access_analysis_target_member.tpl	(.../OpenPNE-2.14.1)	(リビジョン 12654)
@@ -64,6 +64,7 @@
 <input type="hidden" name="month_flag" value="({$month_flag})" />
 <input type="hidden" name="page_name" value="({$requests.page_name})" />
 <input type="hidden" name="orderby" value="({$orderby})" />
+<input type="hidden" name="timestamp" value="({$smarty.now})" />
 <p class="textBtn"><input type="submit" value="ダウンロード" /></p>
 </form>
 ({else})
Index: webapp/modules/admin/templates/access_analysis_target_topic.tpl
===================================================================
--- webapp/modules/admin/templates/access_analysis_target_topic.tpl	(.../OpenPNE-2.14.0)	(リビジョン 12654)
+++ webapp/modules/admin/templates/access_analysis_target_topic.tpl	(.../OpenPNE-2.14.1)	(リビジョン 12654)
@@ -70,6 +70,7 @@
 <input type="hidden" name="month_flag" value="({$month_flag})" />
 <input type="hidden" name="page_name" value="({$requests.page_name})" />
 <input type="hidden" name="orderby" value="({$orderby})" />
+<input type="hidden" name="timestamp" value="({$smarty.now})" />
 <p class="textBtn"><input type="submit" value="ダウンロード" /></p>
 </form>
 ({else})
Index: webapp/modules/admin/templates/csv_download.tpl
===================================================================
--- webapp/modules/admin/templates/csv_download.tpl	(.../OpenPNE-2.14.0)	(リビジョン 12654)
+++ webapp/modules/admin/templates/csv_download.tpl	(.../OpenPNE-2.14.1)	(リビジョン 12654)
@@ -24,6 +24,7 @@
 <input type="hidden" name="start_id" value="0" />
 <input type="hidden" name="end_id" value="0" />
 <input type="hidden" name="allflag" value="1" />
+<input type="hidden" name="timestamp" value="({$smarty.now})" />
 <p class="textBtn"><input type="submit" value="ダウンロード" /></p>
 </form>
 
@@ -35,6 +36,7 @@
 <input type="hidden" name="sessid" value="({$PHPSESSID})" />
 <input class="basic" type="text" name="start_id" value="" size="5" />　～　<input class="basic" type="text" name="end_id" value="" size="5" />
 <input type="hidden" name="allflag" value="0" />
+<input type="hidden" name="timestamp" value="({$smarty.now})" />
 <p class="textBtn"><input type="submit" value="ダウンロード" /></p>
 </form>
 
Index: webapp/modules/admin/templates/user_analysis_date_month.tpl
===================================================================
--- webapp/modules/admin/templates/user_analysis_date_month.tpl	(.../OpenPNE-2.14.0)	(リビジョン 12654)
+++ webapp/modules/admin/templates/user_analysis_date_month.tpl	(.../OpenPNE-2.14.1)	(リビジョン 12654)
@@ -57,6 +57,7 @@
 <input type="hidden" name="m" value="({$module_name})" />
 <input type="hidden" name="a" value="do_({$hash_tbl->hash('csv_user_analysis_date_month','do')})" />
 <input type="hidden" name="sessid" value="({$PHPSESSID})" />
+<input type="hidden" name="timestamp" value="({$smarty.now})" />
 <p class="textBtn"><input type="submit" value="ダウンロード" /></p>
 </form>
 
Index: webapp/modules/admin/templates/delete_topic_comment_selected.tpl
===================================================================
--- webapp/modules/admin/templates/delete_topic_comment_selected.tpl	(.../OpenPNE-2.14.0)	(リビジョン 12654)
+++ webapp/modules/admin/templates/delete_topic_comment_selected.tpl	(.../OpenPNE-2.14.1)	(リビジョン 12654)
@@ -76,7 +76,7 @@
 <tr>
 <th>ファイル</th>
 <td class="textbody">
-<a href="?m=({$module_name})&amp;a=do_({$hash_tbl->hash('file_download','do')})&amp;filename=({$topic_comment.filename})&amp;sessid=({$PHPSESSID})">
+<a href="?m=({$module_name})&amp;a=do_({$hash_tbl->hash('file_download','do')})&amp;filename=({$topic_comment.filename})&amp;sessid=({$PHPSESSID})&amp;({$smarty.now})">
 ({$topic_comment.original_filename})
 </a>
 </td>
Index: webapp/modules/admin/templates/access_analysis_member.tpl
===================================================================
--- webapp/modules/admin/templates/access_analysis_member.tpl	(.../OpenPNE-2.14.0)	(リビジョン 12654)
+++ webapp/modules/admin/templates/access_analysis_member.tpl	(.../OpenPNE-2.14.1)	(リビジョン 12654)
@@ -65,6 +65,7 @@
 <input type="hidden" name="month_flag" value="({$month_flag})" />
 <input type="hidden" name="page_name" value="({$requests.page_name})" />
 <input type="hidden" name="orderby" value="({$orderby})" />
+<input type="hidden" name="timestamp" value="({$smarty.now})" />
 <p class="textBtn"><input type="submit" value="ダウンロード" /></p>
 </form>
 ({else})
Index: webapp/modules/admin/templates/topic_comment_list.tpl
===================================================================
--- webapp/modules/admin/templates/topic_comment_list.tpl	(.../OpenPNE-2.14.0)	(リビジョン 12654)
+++ webapp/modules/admin/templates/topic_comment_list.tpl	(.../OpenPNE-2.14.1)	(リビジョン 12654)
@@ -119,7 +119,7 @@
 <tr>
 <th>ファイル</th>
 <td class="textbody">
-<a href="?m=({$module_name})&amp;a=do_({$hash_tbl->hash('file_download','do')})&amp;filename=({$item.filename})&amp;sessid=({$PHPSESSID})">
+<a href="?m=({$module_name})&amp;a=do_({$hash_tbl->hash('file_download','do')})&amp;filename=({$item.filename})&amp;sessid=({$PHPSESSID})&amp;({$smarty.now})">
 ({$item.original_filename})
 </a>
 </td>
Index: webapp/modules/admin/templates/user_analysis_generation.tpl
===================================================================
--- webapp/modules/admin/templates/user_analysis_generation.tpl	(.../OpenPNE-2.14.0)	(リビジョン 12654)
+++ webapp/modules/admin/templates/user_analysis_generation.tpl	(.../OpenPNE-2.14.1)	(リビジョン 12654)
@@ -50,6 +50,7 @@
 <input type="hidden" name="m" value="({$module_name})" />
 <input type="hidden" name="a" value="do_({$hash_tbl->hash('csv_user_analysis_generation','do')})" />
 <input type="hidden" name="sessid" value="({$PHPSESSID})" />
+<input type="hidden" name="timestamp" value="({$smarty.now})" />
 <p class="textBtn"><input type="submit" value="ダウンロード" /></p>
 </form>
 
Index: webapp/modules/admin/templates/access_analysis_target_diary.tpl
===================================================================
--- webapp/modules/admin/templates/access_analysis_target_diary.tpl	(.../OpenPNE-2.14.0)	(リビジョン 12654)
+++ webapp/modules/admin/templates/access_analysis_target_diary.tpl	(.../OpenPNE-2.14.1)	(リビジョン 12654)
@@ -67,6 +67,7 @@
 <input type="hidden" name="month_flag" value="({$month_flag})" />
 <input type="hidden" name="page_name" value="({$requests.page_name})" />
 <input type="hidden" name="orderby" value="({$orderby})" />
+<input type="hidden" name="timestamp" value="({$smarty.now})" />
 <p class="textBtn"><input type="submit" value="ダウンロード" /></p>
 </form>
 ({else})
Index: webapp/modules/admin/templates/user_analysis_date_day.tpl
===================================================================
--- webapp/modules/admin/templates/user_analysis_date_day.tpl	(.../OpenPNE-2.14.0)	(リビジョン 12654)
+++ webapp/modules/admin/templates/user_analysis_date_day.tpl	(.../OpenPNE-2.14.1)	(リビジョン 12654)
@@ -54,6 +54,7 @@
 <input type="hidden" name="a" value="do_({$hash_tbl->hash('csv_user_analysis_date_day','do')})" />
 <input type="hidden" name="sessid" value="({$PHPSESSID})" />
 <input type="hidden" name="date" value="({$date})" />
+<input type="hidden" name="timestamp" value="({$smarty.now})" />
 <p class="textBtn"><input type="submit" value="ダウンロード" /></p>
 </form>
 
Index: webapp/modules/admin/templates/list_c_file.tpl
===================================================================
--- webapp/modules/admin/templates/list_c_file.tpl	(.../OpenPNE-2.14.0)	(リビジョン 12654)
+++ webapp/modules/admin/templates/list_c_file.tpl	(.../OpenPNE-2.14.1)	(リビジョン 12654)
@@ -60,7 +60,7 @@
 </td>
 ({****})
 <td>
-<a href="./?m=({$module_name})&amp;a=do_({$hash_tbl->hash('file_download','do')})&amp;filename=({$item.filename})&amp;sessid=({$PHPSESSID})">
+<a href="./?m=({$module_name})&amp;a=do_({$hash_tbl->hash('file_download','do')})&amp;filename=({$item.filename})&amp;sessid=({$PHPSESSID})&amp;({$smarty.now})">
 ({$item.filename})
 </a>
 </td>
Index: webapp/modules/ktai/do/o_password_query.php
===================================================================
--- webapp/modules/ktai/do/o_password_query.php	(.../OpenPNE-2.14.0)	(リビジョン 12654)
+++ webapp/modules/ktai/do/o_password_query.php	(.../OpenPNE-2.14.1)	(リビジョン 12654)
@@ -50,7 +50,6 @@
             }
         } else {
             $c_member_id = db_member_c_member_id4ktai_address($ktai_address);
-            // メールアドレスが一致しない場合でも正常に完了した時と同じ画面にする
             if (!$c_member_id) {
                 $p = array('msg' => 26);
                 openpne_redirect('ktai', 'page_o_login', $p);
Index: webapp/modules/ktai/do/h_album_cover_image_delete.php
===================================================================
--- webapp/modules/ktai/do/h_album_cover_image_delete.php	(.../OpenPNE-2.14.0)	(リビジョン 12654)
+++ webapp/modules/ktai/do/h_album_cover_image_delete.php	(.../OpenPNE-2.14.1)	(リビジョン 12654)
@@ -26,7 +26,7 @@
         //---
 
         $c_album_cover = $c_album['album_cover_image'];
-        db_album_image_data_delete($c_album_cover);
+        db_album_image_data_delete($c_album_cover, $u);
         db_album_update_c_album_album_cover_image($target_c_album_id,'');
 
         $p = array('target_c_member_id' => $u, 'target_c_album_id' => $target_c_album_id);
Index: webapp/modules/ktai/do/h_album_image_delete.php
===================================================================
--- webapp/modules/ktai/do/h_album_image_delete.php	(.../OpenPNE-2.14.0)	(リビジョン 12654)
+++ webapp/modules/ktai/do/h_album_image_delete.php	(.../OpenPNE-2.14.1)	(リビジョン 12654)
@@ -35,7 +35,7 @@
         }
         //---
 
-        db_album_delete_c_album_image($target_c_album_image_id);
+        db_album_delete_c_album_image($target_c_album_image_id, $u);
 
         $p = array('target_c_album_id' => $target_c_album_id);
         openpne_redirect('ktai', 'page_fh_album_image_list', $p);
Index: webapp/modules/ktai/do/h_album_cover_image_copy.php
===================================================================
--- webapp/modules/ktai/do/h_album_cover_image_copy.php	(.../OpenPNE-2.14.0)	(リビジョン 12654)
+++ webapp/modules/ktai/do/h_album_cover_image_copy.php	(.../OpenPNE-2.14.1)	(リビジョン 12654)
@@ -39,13 +39,14 @@
         $path_parts = pathinfo($image_filename);
         $ext = $path_parts['extension'];
         $new_filename = 'a_' . $target_c_album_id . '_1_' . time() . '.' . $ext;
-        $c_image_id = db_image_data_copy($image_filename, $new_filename);
+        $c_image_id = db_image_data_copy($image_filename, $new_filename, $u, $target_c_album_image['filesize']);
+
         if (!$c_image_id) {
             openpne_redirect('ktai', 'page_h_err_fh_album');
         }
 
         $c_album_cover = $c_album['album_cover_image'];
-        db_album_image_data_delete($c_album_cover);
+        db_album_image_data_delete($c_album_cover, $u);
         db_album_update_c_album_album_cover_image($target_c_album_id, $new_filename);
 
         $p = array('target_c_member_id' => $u, 'target_c_album_id' => $target_c_album_id);
Index: webapp/version.php
===================================================================
--- webapp/version.php	(.../OpenPNE-2.14.0)	(リビジョン 12654)
+++ webapp/version.php	(.../OpenPNE-2.14.1)	(リビジョン 12654)
@@ -1 +1 @@
-<?php define('OPENPNE_VERSION', '2.14.0'); ?>
+<?php define('OPENPNE_VERSION', '2.15-dev'); ?>
